src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -74,6 +75,16 @@ const useStyles = createUseStyles({
   checkboxMinWidth: {
     minWidth: '45px!important',
   },
+  snapshotInfoText: {
+    color: 'grey',
```

Comment:
So there are colors specific to this from patternfly if you look at the Header.tsx you'll see a similar value (likely to this color) referenced called: 
import { global_Color_100 } from '@patternfly/react-tokens';

global_Color_100.value

Reply:
cool, changed to global_Color_400